src/tests/nativeaot/SmokeTests/PInvoke/PInvoke.cs

```diff
@@ -1053,6 +1105,87 @@ override protected bool ReleaseHandle()
     [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
     public interface IComInterface
     {
+        void DoWork();
+    }
+
+    public class ComObject : IComInterface
+    {
+        public void DoWork()
+        {
+            Console.WriteLine("Do work called.");
```

Comment:
I would add `int` field and int argument for the method, increment the field by the amount passed in here, and then validate in the main test that it was incremented in the main body of the test.